packages/terra-form-select/src/terra-dev-site/doc/form-select/Select.01.doc.mdx

```diff
@@ -47,7 +47,10 @@ This component requires the following peer dependencies be installed in your app
 
 The Form-Select component must be composed inside the [Base][1] component with a locale in order for it to load the correct translation strings.
 
+As a result of accessibility issues, the default implementation of select should no be used. It will be fully removed in a future major version bump. To achieve the same functionality the [Native Select][2] should be consumed.
```

Comment:
```suggestion
As a result of accessibility issues, the default implementation of select should not be used. It will be fully removed in a future major version bump. To achieve the same functionality the [Native Select][2] should be consumed.
```

Reply:
Updated.